x11-wm/openbox/openbox-9999.ebuild

```diff
@@ -41,17 +41,24 @@ RDEPEND="dev-libs/glib:2
 	svg? ( gnome-base/librsvg:2 )
 	xdg? (
 		${PYTHON_DEPS}
-		dev-python/pyxdg[${PYTHON_USEDEP}]
+		$(python_gen_cond_dep '
+			dev-python/pyxdg[${PYTHON_MULTI_USEDEP}]
+		')
 	)
 	"
 DEPEND="${RDEPEND}
 	sys-devel/gettext
 	virtual/pkgconfig
 	x11-base/xorg-proto"
```

Comment:
good opportunity to make closing quotes style consistent, personally I prefer newline + 0 indent